workflows.md

```diff
@@ -0,0 +1,66 @@
+
+## Sigrid CI &amp; multi-branch workflow
+
+**How it works:** In some cases there is no single "main" version of a project, but the development team maintains multiple versions in parallel. For example, a vendor might maintain separate versions for each of their clients. 
+
+<img src="images/multi-branch-workflow.png" width="500" />
+
+**How to use Sigrid CI:** Sigrid does not natively support multiple parallel streams for the same project, since less than 1 percent of our clients use this model. Sigrid assumes a single "baseline" version of the system, which does not exist in this type of workflow. 
+
+However, it is still possible to use Sigrid in such a project: every parallel branch would be a separate system in Sigrid, which allows people to receive feedback for "their" branch. This also means that all branches will be visible as separate systems in the Sigrid dashboard on [sigrid-says.com](https://sigrid-says.com), since they are monitored separately. This in turn allows developers working on different branches to receive Sigrid CI feedback for pull requests targeting their branch.
+
+## Sigrid CI &amp; Git submodule workflow
+
+**How it works:** This workflow does not relate to a particular branching model, but nevertheless influences the development process and how to integrate Sigrid CI. [Git submodules](https://git-scm.com/book/nl/v2/Git-Tools-Submodules) allow a Git repository to include references to other Git repositories. The references act as a pointer, so the submodules retain their own Git commit history that is independent from the parent repository.
+
+<img src="images/git-submodule-workflow.png" width="500" />
+
+**How to use Sigrid CI:** You can use Sigrid CI for projects that use Git submodules. That said, you want to make sure that teams only receive feedback on code they maintain. In the example above, that means the team working on repository B needs to configure Sigrid CI to exclude all code in the Git submodule, since that code is owned by the team that works on repository A.
+
+## Combining multiple repositories into a single Sigrid system
+
+**How it works:** In certain situations, it might be desirable to combine multiple repositories into a single system in Sigrid. This is generally done for reporting purposes: in some projects reporting on every individual repository can seem excessively detailed, and providing a single conclusion simplifies things. 
+
+**How to use Sigrid CI:** Sigrid CI operates on the repository level. This makes sense, as Sigrid CI is integrated into the repository's continuous integration pipeline. However, this leads to a situation where different stakeholders require different views on the source code. 
+
+- For the development teams, use Sigrid CI to provide repository-level feedback as explained in this documentation.
```

Comment:
Otherwise, or maybe regardless of whether we link or not, we could change "this documentation" to "the Frequently Asked Questions".

```suggestion
- For the development teams, use Sigrid CI to provide repository-level feedback as explained in [the Frequently Asked Questions](faq.md#we-have-a-multi-repo-project-can-i-still-use-sigrid-ci).
```